docs/src/stories/components/Alerts/Flash.stories.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import clsx from 'clsx'
+import {InfoIcon, AlertIcon, StopIcon, CheckIcon, XIcon} from '@primer/octicons-react'
+
+export default {
+  title: 'Components/Alerts/Flash',
+  parameters: {
+    // design: {
+    //   type: 'figma',
+    //   url: 'https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=4371%3A7128'
+    // },
+    layout: 'padded'
+  },
+
+  excludeStories: ['FlashTemplate'],
+  argTypes: {
+    variant: {
+      control: 'inline-radio',
+      options: ['info', 'warning', 'error', 'success'],
+    },
+    message: {
+      control: 'text',
+      description: 'The message to alert',
+    },
+    hasVisual: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    hasCloseButton: {
+      control: 'boolean',
+    },
+    isFull: {
+      control: 'boolean',
+    },
+    isFullWhenNarrow: {
+      control: 'boolean',
+    },
+    visualChildren: {
+      description: 'creates a slot for custom visuals',
+    },
+    actionChildren: {
+      description: 'creates a slot for action buttons',
+    }
+  },
+}
+
+export const FlashTemplate = ({
+  variant,
+  message,
+  hasVisual,
+  hasAction,
+  hasCloseButton,
+  isFull,
+  isFullWhenNarrow,
+  visualChildren,
+  actionChildren
+}) => {
+
+  const defaultMessages = {
+    'info': 'This is an info message',
+    'warning': 'This is a warning message',
+    'error': 'This is an error message',
+    'success': 'Your site is published at <a href="#">https://monalisa.github.io/</a>',
+  };
+
+  variant = variant ?? 'info';
+  message = message ?? defaultMessages[variant];
+  hasVisual = hasVisual ?? true;
+
+
+  return (
+    <>
+      <div className={clsx('Flash', variant && `Flash--${variant}`, isFull && `Flash--full`, isFullWhenNarrow && `Flash--full-whenNarrow`)}>
+        {hasVisual && (
+          <div className={clsx('Flash-visual')}>
+            {visualChildren && (
+              <>
+                {visualChildren}
+              </>
+            ) || (
+              <>
+                {variant === 'info' && (<InfoIcon />)}
+                {variant === 'warning' && (<AlertIcon />)}
+                {variant === 'error' && (<StopIcon />)}
+                {variant === 'success' && (<CheckIcon />)}
+              </>
+            )}
+          </div>
+        )}
+
+        <div className={clsx('Flash-message')} dangerouslySetInnerHTML={{__html: message}}></div>
```

Comment:
What do you think about creating a slot for `title` and `description`? The `<strong>` and `<br>` feel like something we should handle at the component level.

Reply:
+1 to what @langermank is suggesting here. I think that this could be similar to the [Blankslate API](https://github.com/github/primer/issues/373#issuecomment-943927091) where both the title and description are styled based on [component classes](https://github.com/primer/view_components/pull/1000).

Reply:
Done!